roles/contiv_network/templates/netplugin.j2

```diff
@@ -0,0 +1,7 @@
+{# If an interface is specified for netplugin, use it #}
+{# Otherwise, hope for the best                       #}
```

Comment:
We should fix existing stuff than leaving it broken as the code has to work correctly in all conditions. If stuff fails, it is an indicator of the fact that those places need to be fixed.

Can we fix this by defining a `netplugin_if` variable (in contiv_network specific vars files) and assigning it a default value like we do for `monitor_interface` value?